src/pages/api/users/index.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { Role } from '../../../enum/role'
+import { user, product } from '../../../entity/entities'
+
+const userHandler = (req: NextApiRequest, res: NextApiResponse) => {
+  const { method, body } = req
+
+  // TODO: Handle the function properly
+  const handlePatch = () => {
+    return {
+      message: 'This is a PATCH request, currently in progress...',
+      context: body
+    }
+  }
+
+  // TODO: Handle the function properly
+  const handlePost = () => {
+    return {
+      users: mockUsers(),
+      context: body
+    }
+  }
+
+  const mockUsers = () => {
+    const userNames: string[] = ['mocking-bird', 'wood-pecker', 'black-crow']
+    const products: Array<ReturnType<typeof product>> = [product('shoe1', 1), product('shoe2', 1), product('shoe3', 1)]
+    const histories: Array<ReturnType<typeof product>> = products.reverse()
+    const users: Array<ReturnType<typeof user>> = userNames.map(name => {
+      const password: string = [...name].reverse().join('')
+      const splitName: string[] = name.split('-')
+
+      return user(name, password, splitName[0], splitName[1], '0', Role.User, products, histories)
```

Comment:
Do you think this is easier to read?
```suggestion
      return user(
        name,
        password,
        splitName[0],
        splitName[1],
        '0',
        Role.User,
        products,
        histories
      )
```

Reply:
No, the amount of letters on the line is not even above 120 letters.